doc/tools/doc_status.py

```diff
@@ -3,18 +3,21 @@
 import fnmatch
 import math
 import os
-import platform
 import re
 import sys
 import xml.etree.ElementTree as ET
 from typing import Dict, List, Set
 
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../"))
```

Comment:
This is so messy :/

I know this was already used in `make_rst.py` but `methods.py` is a collection of methods for use by SCons to build the engine, it's not meant to be used by generic standalone scripts.

Those scripts shouldn't break if we move them around.